sdmetrics/single_table/privacy/dcr_baseline_protection.py

```diff
@@ -48,8 +47,6 @@ def _validate_inputs(
 
         real_data_copy = real_data.copy()
         synthetic_data_copy = synthetic_data.copy()
-        real_data_copy = _process_data_with_metadata(real_data_copy, metadata, True)
-        synthetic_data_copy = _process_data_with_metadata(synthetic_data_copy, metadata, True)
 
         return real_data_copy, synthetic_data_copy, num_rows_subsample, num_iterations
```

Comment:
Since we're no longer processing the data in this method, can we just return the validated `num_rows_subsample` and `num_iterations`? Also, can we add a check that both the real and synthetic dataframe are pandas dataframes?